specs/binding-mqtt.spec/src/main/scripts/io/aklivity/zilla/specs/binding/mqtt/streams/network/v4/publish.one.message.subscribe.unfragmented/client.rpt

```diff
@@ -21,40 +21,28 @@ connect "zilla://streams/net0"
 
```

Comment:
Suggest `.batched` instead of `.unfragmented`.

Also do we need to say `.one.message`? Perhaps `client.publish.subscribe.batched` is sufficient?

Since `.false.start` is specific to `CONNECT` +packet, and this is the more general case, perhaps use `.batched` in place of `.false.start` too, such as `connect.subscribe.batched`?